apps/emqx_management/src/emqx_mgmt_cli.erl

```diff
@@ -576,6 +577,34 @@ data(_) ->
     emqx_ctl:usage([{"data import <File>",   "Import data from the specified file"},
                     {"data export",          "Export data"}]).
 
+%%--------------------------------------------------------------------
+%% @doc acl Command
+
+acl(["cache-clean", "node", Node]) ->
+    case emqx_mgmt:clean_acl_cache_all(erlang:list_to_existing_atom(Node)) of
+        ok ->
+            emqx_ctl:print("ACL cache drain started on node ~s.~n", [Node]);
+        {error, Reason} ->
+            emqx_ctl:print("ACL drain failed on node ~s: ~0p.~n", [Node, Reason])
+    end;
```

Comment:
CLIs don't need to specify the target node, as it implies the command is only sent to the node where you run the command line.